main.py

```diff
@@ -61,7 +61,7 @@
 """
 
 prompt = f"Please extract key information from this email: {email} "
-message = [{"role": "user", "content": prompt}]
+messag = [{"role": "user", "content": prompt}]
```

Comment:
It seems like there's a typo in the variable name. It should be 'message' instead of 'messag'